libraries/testing/src/with-db.js

```diff
@@ -37,11 +37,13 @@ const resetDb = async ({testDbUrl}) => {
   }
 };
 
-const resetTable = async ({testDbUrl, tableName}) => {
+const resetTables = async ({testDbUrl, tableNames}) => {
   const client = new Client({connectionString: testDbUrl});
   await client.connect();
   try {
-    await ignorePgErrors(client.query(`truncate ${tableName}`), UNDEFINED_TABLE);
+    for (let tableName of tableNames) {
+      await ignorePgErrors(client.query(`truncate ${tableName}`), UNDEFINED_TABLE);
+    }
```

Comment:
do I understand correctly the call to `client.connect()` is the slow bit right? Doing these requests serially inside this connection is different than setting up a connection every time?

Reply:
That's my current understanding as well.